site/book-theme/pagetoc.css

```diff
@@ -15,28 +15,33 @@
     .sidetoc {
         position: sticky;
         margin: 0 calc(var(--page-padding) + 15px);
-        width: calc(40% - 15vw + 50px);
+        width: calc(40% - 15vw + 100px);
         max-width: 400px;
         top: calc(var(--menu-bar-height) + 8rem);
         scroll-behavior: smooth;
     }
     .pagetoc {
-        font-size: 95%;
+        font-family: 'Recursive', sans-serif;
+        font-size: 90%;
+        font-weight: 300;
```

Comment:
I prefer the text without these three settings: it's quite hard to read with the other font and when it's thin.